examples/nivisa_dmm_measurement/NIVisaDmmMeasurement.measui

```diff
@@ -1,31 +1,31 @@
 			</ChannelNumericText>
 			<Label Height="[float]16" Id="71a08f4917d143caa1a9edb08cfff3c8" LabelOwner="[UIModel]b8c14ae283ec4bf8ad8a037705e08afc" Left="[float]195" Text="[string]Measured value" Top="[float]10" Width="[float]84" xmlns="http://www.ni.com/PanelCommon" />
-			<ChannelPinSelector AllowUndefinedValues="[bool]True" BaseName="[string]Pin" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Configuration/pin_name" DataType="[Type]String" Height="[float]24" Id="f7b79f2403f9467782561c9c75a54604" IsLabelBoundToChannel="[bool]False" Label="[UIModel]9905796649f44175a56f9eb34178d24e" Left="[float]23" SelectedResource="[NI_Core_DataValues_TagRefnum]Pin1" Top="[float]30" Width="[float]150" xmlns="http://www.ni.com/InstrumentFramework/ScreenDocument" />
+			<ChannelPinSelector AllowUndefinedValues="[bool]True" BaseName="[string]Pin" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Configuration/pin_name" DataType="[Type]String" Enabled="[bool]True" Height="[float]24" Id="f7b79f2403f9467782561c9c75a54604" IsLabelBoundToChannel="[bool]False" Label="[UIModel]9905796649f44175a56f9eb34178d24e" Left="[float]23" SelectedResource="[NI_Core_DataValues_TagRefnum]Pin1" Top="[float]30" Width="[float]150" xmlns="http://www.ni.com/InstrumentFramework/ScreenDocument" />
 			<Label Height="[float]16" Id="9905796649f44175a56f9eb34178d24e" LabelOwner="[UIModel]f7b79f2403f9467782561c9c75a54604" Left="[float]23" Text="[string]Pin name" Top="[float]10" Width="[float]49" xmlns="http://www.ni.com/PanelCommon" />
-			<ChannelEnumSelector AdaptsToType="[bool]True" AllowNonSequentialValues="[bool]True" BaseName="[string]Enum" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Configuration/measurement_type" Height="[float]24" Id="dada7f23e27f4f468c0faf48553a7b66" IsLabelBoundToChannel="[bool]False" Label="[UIModel]570f36c6f42d4eb6a90f7e8a826ab54d" Left="[float]23" Top="[float]85" Width="[float]150" xmlns="http://www.ni.com/InstrumentFramework/ScreenDocument">
-				<RingSelectorInfo DisplayValue="[string]DC_VOLTS" IsEnabled="[bool]True" Value="[int]1" ValueType="[Type]Int32" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
-				<RingSelectorInfo DisplayValue="[string]AC_VOLTS" IsEnabled="[bool]True" Value="[int]2" ValueType="[Type]Int32" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
+			<ChannelEnumSelector AdaptsToType="[bool]True" AllowNonSequentialValues="[bool]True" BaseName="[string]Enum" Channel="[string]{26c56ad8-e31a-456d-979e-87865b3f5c61}/Configuration/measurement_type" Enabled="[bool]True" Height="[float]24" Id="dada7f23e27f4f468c0faf48553a7b66" IsLabelBoundToChannel="[bool]False" Label="[UIModel]570f36c6f42d4eb6a90f7e8a826ab54d" Left="[float]23" Top="[float]85" Width="[float]150" xmlns="http://www.ni.com/InstrumentFramework/ScreenDocument">
+				<RingSelectorInfo DisplayValue="[string]DC_VOLTS" IsEnabled="[bool]True" Value="[int]0" xmlns="http://www.ni.com/Controls.LabVIEW.Design" />
```

Comment:
Hmm, this must be from an earlier version of the service that didn't have a 0 enum value.

Reply:
Yeah I wasn't sure about this change.

Reply:
Is there any action I need to take here?

Reply:
I don't think so.